### PR TITLE
added fxn that will return three random keywords and pass down to Gip…

### PR DESCRIPTION
**Pull request by jhshim85**
added fxn that will return three random keywords from movie api and pass to giphy component
took out -1 in randomizer fxn

### CODE REVIEW

File: src/Form.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const Form = () => {
             console.log(res.data.results[0]);
```
Comment (beatricegeorgeslemire) [inline src/Form.js:27]: It looks great, and makes sense! Thank you Jay!

File: src/Form.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ const Form = () => {
...
             setMovieId(res.data.results[0].id);
         })
+        // reset the state of userInput and movieID to an empty string
```
Comment (karleene) [inline src/Form.js:30]: Thank you for adding pseudo codes!

File: src/MovieApi.js [hunk 3]
```diff
@@ -17,12 +20,14 @@ const MovieApi = (props) => {
...
             })
+            // reset the state of userInput and movieID to an empty string
             setMovieKeyword([])
         }
 }, [props.movieId])
 
     return (
-        movieKeyword.slice(0,3).map((keyword)=>{
+        // passing only 3 keywords randomly to GiphyApi component
+        movieKeyword.sort(() => 0.5 - Math.random()).slice(0,3).map((keyword)=>{
```
Comment (karleene) [inline src/MovieApi.js:30]: Good job! Thanks for doing this. I would love to hear you explain sort and why it's 0.5. I need help understanding it better

**Comment by jhshim85** [comment]
making new pull request with fixed error

**Pull request closed without merging**